ui/src/app/shared/history/history.html

```diff
@@ -37,14 +37,14 @@
                 <a href="#"
                    [routerLink]="['/project', project.key, 'application', pb.application.name, 'pipeline', pb.pipeline.name, 'build', pb.build_number]"
                    [queryParams]="{ envName: pb.environment.name, branch: pb.trigger.vcs_branch, version: pb.version}">
-                    {{pb.start | amCalendar}}
+                    {{pb.start | amCalendar:{'sameElse': 'MM/DD/YYYY at h:mm A'} }}
```

Comment:
Have you take a look to the date pipe from angular that take care of local  fr/en?

Reply:
DD/MM/YYYY/ => MM/DD/YYYY    No. Must depend of language